test/automation/src/terminal.ts

```diff
@@ -82,7 +85,7 @@ export class Terminal {
 		// new quick input shows.
 		await this.quickinput.waitForQuickInputOpened();
 		if (value) {
-			await this.code.waitForSetValue(QuickInput.QUICK_INPUT_INPUT, value);
```

Comment:
Should these static members be private to avoid outside usage?

Reply:
good point, via https://github.com/microsoft/vscode/pull/139367/commits/a516a34f88ee344257859fe2b9f0ea2384b0c534